### DISCUSSION TITLE
border radius

### DISCUSSION THREAD

**Discussion by DeekshantSharma16** [yifaneye/react-gallery-carousel#66] [Q&A] [answered]
I am not able to give border radius in this how to solve this ?

**Comment by yifaneye** [comment, marked as answer]
Please try:
```
<Carousel
    images={images}
    style={{ borderRadius: 10, overflow: 'hidden' }}
/>
```

**Reply by yifaneye** [reply]
Another example is:
https://github.com/yifaneye/react-gallery-carousel/blob/990f45500c60e3926702538bd9c61dacc681948b/example/src/components/Carousel4.js#L32
https://github.com/yifaneye/react-gallery-carousel/blob/990f45500c60e3926702538bd9c61dacc681948b/example/src/index.css#L100
